client/src/SignupNonProfit.tsx

```diff
@@ -0,0 +1,191 @@
+import * as React from 'react';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+
+function SignupNonProfit() {
+    interface SignupData {
+        org_name: string;
+        city: string;
+        state: string;
+        ein: string;
+        tax_exempt_id: string;
+        nonprofit_classification: string;
+        first_name: string;
+        last_name: string;
+        role_or_title: string;
+        email: string;
+        password: string;
+        accept_terms: string;
+    }
+
+    const initialFormData: SignupData = {
+        org_name: '',
+        city: '',
+        state: '',
+        ein: '',
+        tax_exempt_id: '',
+        nonprofit_classification: '',
+        first_name: '',
+        last_name: '',
+        role_or_title: '',
+        email: '',
+        password: '',
+        accept_terms: 'false'
+    };
+
```

Comment:
move static content out of component

Reply:
I'll update the PR with this one before merging, since I am not currently working on that component.